jib-core/src/test/java/com/google/cloud/tools/jib/cache/LayerEntriesSelectorTest.java

```diff
@@ -52,7 +51,15 @@
       ImmutableList<LayerEntry> layerEntries) {
     return layerEntries
         .stream()
-        .map(layerEntry -> new LayerEntryTemplate(layerEntry, Instant.EPOCH.toString()))
+        .map(
+            layerEntry -> {
+              try {
```

Comment:
For this kind of method, should probably just use an iterative approach over stream to avoid the try-catch here.